tests/qchem/test_factorization.py

```diff
@@ -114,12 +114,11 @@ def test_factorize(two_tensor, tol_f, tol_s, factors_ref):
 )
 def test_factorize_reproduce(two_tensor):
     r"""Test that factors returned by the factorize function reproduce the two-electron tensor."""
-    factors, _, _ = qml.qchem.factorize(two_tensor, 1e-5, 1e-5)
-    two_computed = np.zeros(two_tensor.shape)
-    for mat in factors:
-        two_computed += np.einsum("ij, lk", mat, mat)
+    factors1, _, _ = qml.qchem.factorize(two_tensor, 1e-5, 1e-5, cholesky=False)
```

Comment:
Do we only need to test the factors for correction and not the eigenvalues and eigenvectors?